src/test/scala/higherkindness/compendium/http/RootServiceSpec.scala

```diff
@@ -255,7 +255,6 @@ object RootServiceSpec extends Specification with ScalaCheck {
           )
         )
       val response = RootService.rootRouteService[IO].orNotFound(request).unsafeRunSync
-
       response.status == Status.BadRequest
     }
```

Comment:
I have a question about this, @fedefernandez.
Since now the `ValidationParam` it's an `OptionalQueryParam` I guess we're not validating that it's something different than  `true/false` (in that case it's evaluated like `None`, and then default case it's done). Is it like this? Should I remove in that case this test?